src/Faithlife.Tracing.AspNet/NullTraceProvider.cs

```diff
@@ -0,0 +1,12 @@
+namespace Faithlife.Tracing.AspNet
+{
+	/// <summary>
+	/// An <see cref="ITraceProvider"/> that returns <c>null</c> for <c>CurrentTrace</c>.
+	/// </summary>
+	internal sealed class NullTraceProvider : ITraceProvider
```

Comment:
I assumed we would put this in `Faithlife.Tracing` and make it `public`.

Reply:
That would make more sense, since `NullTraceProvider` is not just for ASP.NET.